src/photos/components/PhotoList.jsx

```diff
@@ -13,6 +13,11 @@ const photoDimensionsFallback = { width: 1, height: 1 }
 const adaptRowHeight = containerWidth => 180 + (containerWidth || 1800) / 30
 
 export class PhotoList extends Component {
+  isPhotoSelected(photo, selected) {
+    const selectedIds = selected.map(p => p._id)
+    return selectedIds.indexOf(photo._id) !== -1
+  }
```

Comment:
```js
isPhotoSelected(photo, selected) {
   return !! selected.find(p => (p._id == photo._id));
}
```

not tested but probably better since it does not require a full array with all selected ids and then a search inside this array